docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -852,16 +852,42 @@ multiFuzzyMatchAllIndices(haystack, distance, \[pattern<sub>1</sub>, pattern<sub
 
 ## extract
 
-Extracts a fragment of a string using a regular expression. If `haystack` does not match the `pattern` regex, an empty string is returned.
-
-For regex without subpatterns, the function uses the fragment that matches the entire regex. Otherwise, it uses the fragment that matches the first subpattern.
+Extracts a fragment of a string using a regular expression.
```

Comment:
Sounds slightly odd. Maybe `Returns the first match of a regular expression in a string`? (please also adjust l. 893)